### DISCUSSION TITLE
Asterisks when typing password in SSH/su/login shell

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#148] [Ideas]
You already modify sudo to show asterisks, can you do that for `su`, SSH and the login shell?

**Comment by rcalixte** [comment]
The sudo change is a built-in sudoers configuration option (`pwfeedback`) but the other items in your list either don't currently support that kind of option or it was removed at some point in the past due to being technically less secure.

This means either those packages would either need to be forked or pull requests would need to be submitted ***and accepted*** upstream. Speaking for myself, both routes are unlikely endeavors.
